analytics.proto

```diff
@@ -18,7 +18,7 @@ enum StreamType {
 }
 
 message AnalyticsStat {
-  string auth_token = 1;
+  string access_key = 1;
```

Comment:
main concern with this name is if it'll be confusing compared to our `access token`. would it be better calling this the `analytics key` ?